src/scripts/modules/media/body/content-baked.less

```diff
@@ -178,8 +185,6 @@ h1.example-title .text {
 .note     { .make-block(note); }
 [data-type="example"],
 .example  { .make-block(example); }
-[data-type="abstract"],
```

Comment:
abstract = chapter outline? does it need to happen globally?
if that code doesn't affect anything else badly, I approve